src/libraries/System.Collections/src/System/Collections/BitArray.cs

```diff
@@ -910,6 +910,56 @@ public unsafe void CopyTo(Array array, int index)
             }
         }
 
+        public bool HasAllSet()
```

Comment:
Please add XML comments documenting the methods.

Reply:
Added.